modules/building/pages/activation-keys-subscription.adoc

```diff
@@ -212,25 +222,33 @@ NOTE: For this step we will assume that you have source code in your current wor
 
 In this example, we'll using the Red Hat Enterprise Linux 9 Universal Base Image (UBI 9).
 
++
```

Comment:
This seems to render as a literal +

![image](https://github.com/user-attachments/assets/2cca90e5-231a-4bbb-a10b-15d0ccfd7f53)